examples/manifold/plot_compare_methods.py

```diff
@@ -43,14 +43,10 @@
 plt.suptitle("Manifold Learning with %i points, %i neighbors"
              % (1000, n_neighbors), fontsize=14)
 
-try:
-    # compatibility matplotlib < 1.0
-    ax = fig.add_subplot(251, projection='3d')
-    ax.scatter(X[:, 0], X[:, 1], X[:, 2], c=color, cmap=plt.cm.Spectral)
-    ax.view_init(4, -72)
-except:
-    ax = fig.add_subplot(251, projection='3d')
-    plt.scatter(X[:, 0], X[:, 2], c=color, cmap=plt.cm.Spectral)
+
+ax = fig.add_subplot(251, projection='3d')
```

Comment:
If I believe the comment it looks like you kept the matplotlib < 1 code. The comment may have been misplaced. You should double-check that view_init does not work for matplotlib < 1. Double-check for your other changes too.